components/patient-data/pageobjects/src/main/java/org/phenotips/data/test/po/PatientRecordEditPage.java

```diff
@@ -955,11 +957,11 @@ public boolean checkIfClinicalSymptomsAndPhysicalFindingsExpanded(By by)
     public void selectPhenotype(String id, boolean positive)
     {
         BaseElement
-            .getUtil()
-            .findElementWithoutWaiting(
-                getDriver(),
-                By.cssSelector("label[for='PhenoTips.PatientClass_0_" + (positive ? "" : "negative_") + "phenotype_"
-                    + id + "']")).click();
+        .getUtil()
+        .findElementWithoutWaiting(
+            getDriver(),
+            By.cssSelector("label[for='PhenoTips.PatientClass_0_" + (positive ? "" : "negative_") + "phenotype_"
+                + id + "']")).click();
```

Comment:
Lots of whitespace-only changes here. I think ideally this should be avoided in the future because it inflates the diff and makes the pull request more difficult to review.